examples/plot_stack_masks.py

```diff
@@ -0,0 +1,131 @@
+"""
+Mask and Plot Remote Sensing Data with EarthPy
+==============================================
+
+
+"""
+
+###############################################################################
+# Plotting with EarthPy
+# ---------------------
+#
+# .. note:: Below we walk through a typical workflow using Landsat data with EarthPy.
+
+
+###############################################################################
+# Import Packages
+# ------------------------------
+#
+# To begin, import the needed packages. You will use a combination of several EarthPy
+# modules including spatial, plot and mask.
+
+from glob import glob
+import os
+import matplotlib.pyplot as plt
+import rasterio as rio
+from rasterio.plot import plotting_extent
+import earthpy as et
+import earthpy.spatial as es
+import earthpy.plot as ep
+import earthpy.mask as em
+
+# Get data and set your home working directory
+data = et.data.get_data("cold-springs-fire")
+
+###############################################################################
+# Import Example Data
+# ------------------------------
+# To get started, make sure your directory is set. Create a stack from all of the
+# Landsat .tif files (one per band) and import the ``landsat_qa`` layer which provides
+# the locations of cloudy and shadowed pixels in the scene.
+
+os.chdir(os.path.join(et.io.HOME, "earth-analytics"))
```

Comment:
Changing directories like this seems like bad practice - and makes a very strong assumption about the user's filesystem. The object `data` defined above should provide the paths to any downloaded files - you can probably use that object to get file paths instead of changing directories.

Reply:
we could but that is how we've done it in ALL of our lessons. so we are talking about a hundred or so lessons atleast?? the idea was to ensure the users all have the same paths so the lesson code works for them. what would be the alternative here? to fix every path in all of the lessons and remove the change dir code that we've used? it's just a fairly big lift to do that.

i suggest this is another issue rather than something i have to manage in this PR to add vignettes.

Reply:
I agree we should probably figure this out elsewhere - and it's definitely a longer term discussion point since there are so many tutorials that are doing this.